### PR TITLE
Reverses the order of information presented in the Graphical interfac…

### PR DESCRIPTION
**Pull request by bredamc**
…es examples

### CODE REVIEW

**Comment by bredamc** [comment]
@ndeevy Can you please review to make sure I've created the PR correctly? Many thanks :)

File: supplementary_style_guide/style_guidelines/graphical-interfaces.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ Bold all graphical user interface elements, including, but not limited to, menu
 
 .Example AsciiDoc
 ----
-Click *Metering* on the *Installed Operators* page.
+On the *Installed Operators* page, click *Metering* .
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/graphical-interfaces.adoc:14]: There is an extra space before the ending period, if you can remove that

**Review by bergerhoffer — COMMENTED** [review]
@bredamc Thanks for opening this PR! I see just one minor issue if you're able to update, then this LGTM!

**Comment by ndeevy** [comment]
@bredamc  Looks great to me! I'll hit the approve button now :-]

**Review by ndeevy — APPROVED** [review]

**Review by bergerhoffer — APPROVED** [review]
LGTM!

@redhat-documentation/ccs-style-council Needs one more approval please :)

**Comment by sterobin** [comment]
Approved. Looks good. Thanks @bredamc !

**Comment by bergerhoffer** [comment]
Thanks all! With 3 approvals, I'm merging this in.

**Pull request merged**